base/setup/lib/muifonts.h

```diff
@@ -388,7 +384,7 @@ MUI_SUBFONT HindiFonts[] =
     { L"Courier New Greek,161",  L"Courier New,161" },
     { L"Courier New TUR,162",    L"Courier New,162" },
     { L"Fixedsys",               L"Fixedsys Excelsior 3.01-L2" },
-    { L"Helv",                   L"Tahoma" },
+    { L"Helv",                   L"MS Sans Serif" },
     { L"Helvetica",              L"Arial" },
     { L"MS Sans Serif",          L"FreeSans" },
```

Comment:
Double-checking: what about this substitute? (Helv: Tahoma vs FreeSans...)

Reply:
Helv is Win16-compatible font substitute that is mapped to MS Sans Serif in Windows. Some languages are lacking some language-specific glyphs in MS Sans Serif.